draft-ietf-ppm-dap.md

```diff
@@ -968,7 +960,8 @@ This message consists of:
 * Information used by the Aggregators to determine how to aggregate each report:
 
     * For fixed-size tasks, the Leader specifies a "batch ID" that determines
-      the batch to which each report for this aggregation job belongs.
+      the batch to which each report for this aggregation job belongs. The
+      Leader MUST allocate these IDs consecutively starting from 0.
```

Comment:
this contradicts with previous text: "it is expected that the batch ID to start
 from will be configured out-of-band from the DAP protocol." at line #571

Reply:
Ah, good point. The intent of the text you reference on line 571 is that a Collector that starts after the Aggregators have been running may not be able to start from 0 -- while the Aggregators generated batch IDs starting form 0, by the time the Collector starts, batch ID 0 will have already been garbage collected & no longer be available for collection. Given that, the Collector needs some way to know where to start from.

That said, this text is confusing & likely unnecessary at this point. I deleted the relevant text around line 571.